### PR TITLE
Manual submission (v2) 

### PR DESCRIPTION
**Pull request by Sumukh**

### CODE REVIEW

**Comment by knrafto** [comment]
Tried uploading and autograding a hog.py under Cale's email.

I get this popup:
```
The autograder rejected your request. <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

**Comment by Sumukh** [comment]
@knrafto Can you try again now? Just pushed a new version to staging with the fix.

File: server/app/api.py [hunk 1]
```diff
@@ -920,7 +920,7 @@ def manual_submit(self, obj, user, data):
         if not submitter:
             raise BadValueError('User does not exist')
 
-        files = {f.filename: f.getvalue().decode('latin-1')
+        files = {f.filename:f.getvalue().decode('latin-1')
```
Comment (knrafto) [inline server/app/api.py]: Why latin-1 instead of utf-8?

**Comment by knrafto** [comment]
Okay, it submitted successfully, but the autograder still has not scored it. I don't know whether it failed or it's just slow

**Comment by knrafto** [comment]
LGTM

**Pull request merged**